src/components/RightSidebar/Matterbridge/MatterbridgeSettings.vue

```diff
@@ -565,38 +622,118 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-.loading {
-	margin-top: 30px;
+::v-deep .icon-multiselect-service {
+	width: 16px !important;
+	height: 16px !important;
+	margin-right: 10px;
+}
+
+body.theme--dark .icon-multiselect-service {
+	-webkit-filter: invert(1);
+	filter: invert(1);
 }
 
-#matterbridge-header {
-	padding-left: 40px;
-	padding-top: 40px;
+.matterbridge-settings {
+	.loading {
+		margin-top: 30px;
+	}
 
 	h3 {
 		font-weight: bold;
+		padding: 0;
+		height: 44px;
+		display: flex;
+
+		p {
+			margin-top: auto;
+			margin-bottom: auto;
+		}
+
+		.icon {
+			display: inline-block;
+			width: 40px;
+		}
+		&:hover {
+			background-color: var(--color-background-hover);
+		}
 	}
 
-	p {
-		color: var(--color-text-maxcontrast);
+	#matterbridge-header {
+		padding: 0 0 10px 0;
+
+		p {
+			padding-left: 40px;
+			color: var(--color-text-maxcontrast);
+
+			a:hover,
+			a:focus {
+				border-bottom: 2px solid var(--color-text-maxcontrast);
+			}
 
-		a:hover,
-		a:focus {
-			border-bottom: 2px solid var(--color-text-maxcontrast);
+			a .icon {
+				display: inline-block;
+				margin-bottom: -3px;
+			}
 		}
 	}
-}
 
-.basic-settings {
-	button,
-	.multiselect {
-		width: calc(100% - 40px);
-		margin-left: 40px;
+	.basic-settings {
+		.action {
+			list-style: none;
+		}
+		.save-changes {
+			width: 100%;
+			text-align: left;
+
+			.icon-checkmark {
+				margin: 0 10px 0 2px;
+			}
+		}
+		.multiselect {
+			width: calc(100% - 44px);
```

Comment:
```suggestion
			width: calc(100% - 46px);
```
seems to fix the dyslexia